include/mfmg/dealii/amge_host.hpp

```diff
@@ -41,16 +42,35 @@ class AMGe_host : public AMGe<dim, VectorType>
    * the diagonal elements of the local system matrix, and a vector that maps
    * the dof indices from the local problem to the global problem.
    */
+  template <typename Triangulation>
```

Comment:
We need a comment here so we don't forget why we template on `Triangulation`

Reply:
Done